templates/circleci-2.0/config.yml

```diff
@@ -4,12 +4,9 @@
 #   1. Copy the .circleci directory to the module.
 #   2. Update the working_directory key.
 #   3. Update the module path in the code-sniffer job.
-#   4. If your module depends on a private Composer repository, add it to each
```

Comment:
It was nice discovering that the composer merge plugin does repositories for us automatically.

Reply:
What do you mean?

Reply:
By default, it merges repositories from merged composer.json files. That means we don't need to manually configure it, which saves us this setup complexity. Winning!